File: VB/WindowsFormsApplication79/CustomTileNavPane.vb
Imports DevExpress.XtraBars.Navigation
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication79
    Public Class CustomTileNavPane
        Inherits TileNavPane

        Private _arrowGlyph As Bitmap
        Public Sub New()
            MyBase.New()

        End Sub
        Protected Overrides Function CreateViewInfo() As TileNavPaneViewInfo
            Return New CustomTileNavPaneViewInfo(Me)
        End Function
        Public Property ArrowGlyph() As Bitmap
            Get
                Return _arrowGlyph
            End Get
            Set(ByVal value As Bitmap)
                _arrowGlyph = value
            End Set
        End Property
    End Class
End Namespace

File: CS/WindowsFormsApplication79/CustomTileNavPane.cs
using DevExpress.XtraBars.Navigation;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication79 {
    public class CustomTileNavPane : TileNavPane {
        Bitmap _arrowGlyph;
        public CustomTileNavPane() : base() {

        }
        protected override TileNavPaneViewInfo CreateViewInfo() {
            return new CustomTileNavPaneViewInfo(this);
        }
        public Bitmap ArrowGlyph { get { return _arrowGlyph; } set { _arrowGlyph = value; } }
    }
}
